common/api/imodeljs-frontend.api.md

```diff
@@ -235,6 +235,11 @@ import { Range3d } from '@bentley/geometry-core';
 import { Range3dProps } from '@bentley/geometry-core';
 import { Ray3d } from '@bentley/geometry-core';
 import { ReadonlySortedArray } from '@bentley/bentleyjs-core';
+import { RealityData } from '@bentley/reality-data-client';
+import { RealityDataFormat } from '@bentley/imodeljs-common';

```

Comment:
Combine imports from same package into a single import statement.

Reply:
This file cannot be edited. It is a report generated by [API Extractor](https://api-extractor.com/).
My netries cannot be combine and I don't knwo about the others